src/daemon-commander.ts

```diff
@@ -20,7 +20,15 @@ import type { Response } from 'got';
 import got from 'got';
 import { isWindows } from './util';
 
-export type CrcStatus = 'Running' | 'Starting' | 'Stopping' | 'Stopped' | 'No Cluster' | 'Error' | 'Unknown';
+export type CrcStatus =
+  | 'Running'
+  | 'Starting'
+  | 'Stopping'
+  | 'Stopped'
+  | 'No Cluster'
+  | 'Error'
+  | 'Unknown'
+  | 'Need Setup';
```

Comment:
this can also be used to set the tray to show a Setup option instead of Start/Stop when not finished yet.